src/status_im/ui2/screens/chat/composer/view.cljs

```diff
@@ -188,7 +188,7 @@
                                                        (re-frame/dispatch [:chat.ui/send-current-message]))}
                     :i/arrow-up]]])
                ;black background
-               [reanimated/view {:style (reanimated/apply-animations-to-style
-                                         {:opacity bg-opacity}
-                                         (styles/bottom-sheet-background window-height))}]
+               #_[reanimated/view {:style (reanimated/apply-animations-to-style
```

Comment:
why not fix it?

Reply:
just to unblock chat for now? ok then we need to create an issues so we can fix it properly

Reply:
@flexsurfer I don't mind fixing it, currently I'm not sure where this background is used and how to check whether it works at all

Reply:
yeah it is used when you maximize the composer, you can pull it, or type lots of text

Reply:
ok, I'll take a look then, thanks